_includes/footer.html

```diff
@@ -1,7 +1,7 @@
 
 <footer role="contentinfo" class="site-footer">
 
-	<nav>
+	<nav class="site-footer__navigation">
 		<ul class="site-navigation">
```

Comment:
Based on Slack discussion (and that this is hopefully getting removed by whoever designs the site), I'm gonna leave it.